.github/workflows/tests.yml

```diff
@@ -260,8 +252,8 @@ jobs:
           environment-file: ci/environment-dashboard.yml
 
       - name: Generate dashboards
-        # Show differences of upstream vs. latest unpublished vs. latest publshed
-        run: python dashboard.py -d benchmark.db -o static -b coiled-latest-py3.9 coiled-0.2.1-py3.9
+        # Show differences of upstream vs. latest unpublished
+        run: python dashboard.py -d benchmark.db -o static -b coiled-latest-py3.9
```

Comment:
I think this change is okay from skimming `dashboard.py`, but @crusaderky could you confirm?